install/installer/pkg/components/ws-daemon/constants.go

```diff
@@ -11,7 +11,6 @@ const (
 	ServicePort             = 8080
 	HostWorkingArea         = "/var/gitpod/workspaces"
```

Comment:
is this still used somewhere or can we remove it?

Reply:
Good catch! I do not see it used anywhere.